File: src/beginner/day09/Question01.java
package beginner.day09;

public class Question01 {

    public static class Solution {

        public int solution(int hp) {

            return (hp / 5) + (hp % 5) / 3 + ((hp % 5) % 3);
        }
    }

    public static void main(String[] args) {

        Question01.Solution solution = new Solution();

        System.out.println(solution.solution(23));
        System.out.println(solution.solution(24));
        System.out.println(solution.solution(999));
    }
}

File: src/beginner/day21/Question01.kt
package beginner.day21

class Question01{

    fun solution(my_string: String): Int {
        var answer: Int = 0

        val delimiter = Regex("[a-zA-Z]")

        val strs = my_string.split(delimiter)

        for (str in strs) {
            if (str != "") answer += str.toInt()
        }

        return answer
    }
}

fun main() {

    val solution = Question01()

    println(solution.solution("aAb1B2cC34oOp"))
    println(solution.solution("1a2b3c4d123Z"))
}